socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
-            )
+            self.logger.debug('writing to %s' % (
+                hasattr(a_store, 'wrapped_object') and
```

Comment:
Don't use `hasattr`. https://hynek.me/articles/hasattr/

You can do this line instead with `getattr(a_store, 'wrapped_object', a_store.__class__)`.

By the way, shouldn't it be `a_store.__class__.__name__`? If `a_store` has a `__str__` method, you'd execute that, when in fact, I think you want to log the name of the class.